pyanaconda/anaconda_log.py

```diff
@@ -32,6 +32,10 @@
 STDOUT_FORMAT = "%(asctime)s %(message)s"
 DATE_FORMAT = "%H:%M:%S"
 
+# the Anaconda log uses structured logging
+ANACONDA_ENTRY_FORMAT = "%(asctime)s,%(msecs)03d %(levelname)s %(log_prefix)s: %(message)s"
```

Comment:
Yeah, there is a lot of stuff to log - and especially stuff like path to file where the log message originated from or the line number could certainly be useful for debugging. But at the same time I'm not sure how readable the log file would be in such a case.

So I'm thinking about two things that could be done:
- we can do a detailed/verbose (`anaconda-verbose.log`, etc.) log that is created if inst.debug=1 is used, which h could contain path to file, function name and line number
  - like this we still have a readable `anaconda.log` for general debugging and the new detailed log for detailed debugging
- if we switched from using the SyslongHandler to actual Journal handler for forwarding of Anaconda log messages to journal, all the extra data fields should end up in Journal as well
  - we might need to change our log grabbing procedures to either grab the full journal or some expert that has the extra fields
  - we might also want to investigate how journalctl can present the extra fields in it's output, as just displaying all of them at once would make the log mostly unreadable
  - I guess that the extra data fields can be useful for kickstart tests and other forms of testing